index.html

```diff
@@ -1343,42 +1355,50 @@
           </p>
           <p>
             When the <code><dfn>close</dfn>()</code> method is called on a
-            <a>PresentationConnection</a>, the user agent MUST run the
-            algorithm to <a data-lt="close-algorithm">close a presentation
-            connection</a> with <a>PresentationConnection</a>, a
-            <code>reason</code> of <code>closed</code>, and an empty
-            <code>message</code>.
+            <a>PresentationConnection</a> <var>S</var>, the <a>user agent</a>
+            MUST <a>start closing the presentation connection</a> <var>S</var>
+            with <code>closed</code> as <code>closeReason</code> and an empty
+            <code>message</code> as <code>closeMessage</code>.
```

Comment:
I agree that cases 1 and 3 need to be addressed.  I will fix the PR to ensure that the connection state is set to 'closed' on entry to either the "starting" or "closing" procedures which should fix the re-entrancy cases.

I don't see a scenario where 2 should be possible (if the object is discarded by definition there should be no live references to it).